content/open-source/sigstore/policy-controller/how-to-install-policy-controller.md

```diff
@@ -1,5 +1,7 @@
 ---
```

Comment:
not sure how this got missed in the audit but this doc has a section that points to Enforce docs which we ought to remove: https://deploy-preview-1558--ornate-narwhal-088216.netlify.app/open-source/sigstore/policy-controller/how-to-install-policy-controller/#options-for-continuous-verification

Reply:
Ah, I see now that you removed this same section from several guides below. Well here's another one that ought to get cut